metadata.yml

```diff
@@ -1,3 +1,8 @@
 name: "keystone"
 version: "2016.4.1"
-source: "https://github.com/openstack/salt-formula-keystone"
+source: "https://github.com/salt-formulas/salt-formula-keystone"
+dependencies:
```

Comment:
ok,

NOTE: you should consider to have in metadata.yml section for like mandatory and also optional dependencies (with "optional" as a flag.)